### PR TITLE
draft of CFN lesson without images

### PR DESCRIPTION
**Pull request by ElviraQDP**
missing images

### CODE REVIEW

File: src/content/lesson/[draft]crow-foot-notation.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
++ one or many
+
+![Crow's foot notation](../../assets/images/image-7-cfn.png)
+
++ one and only one
+
+![Crow's foot notation](../../assets/images/image-8-cfn.png)
+
++ zero or one:
```
Comment (marcogonzalo) [inline src/content/lesson/[draft]crow-foot-notation.md:103]:
```suggestion
+ zero or one
```

File: src/content/lesson/[draft]crow-foot-notation.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+![Crow's foot notation](../../assets/images/image-13-cfn.png)
+
+You can see from this image just where the name "crow's foot" came from...
+
++ An optional relationship is represented by an empty circle.
+
+Finally! There are 4 possible edges to the relationship, illustrated here:
+
++ zero or many:
```
Comment (marcogonzalo) [inline src/content/lesson/[draft]crow-foot-notation.md:91]:
```suggestion
+ zero or many
```

**Review by marcogonzalo — CHANGES_REQUESTED** [review]

**Comment by alesanchezr** [comment]
Change wording to avoid copy-cat
First paragraph its duplciated.

**Review by JimenaEB — CHANGES_REQUESTED** [review]

**Review by ElviraQDP — COMMENTED** [review]
Que falta corregir @marcogonzalo ?

**Pull request closed without merging**